ferrocene/doc/sphinx-shared-resources/src/ferrocene_qualification/__init__.py

```diff
@@ -56,8 +56,7 @@ def validate_config(app, config):
 
 
 def inject_version(app, config):
```

Comment:
is this fn used anywhere else... asking since it was introduced with addition of sphinx-needs support